src/lib/linux-api/src/time.rs

```diff
@@ -32,23 +33,25 @@ pub enum ClockId {
 }
 
 pub use bindings::linux_timespec;
-pub type TimeSpec = linux_timespec;
+#[allow(non_camel_case_types)]
+pub type timespec = linux_timespec;
```

Comment:
maybe make this Pod, and then can use the pod zero below instead of the unsafe zero?

Reply:
done